docs/make.jl

```diff
@@ -0,0 +1,21 @@
+using Documenter
+using FastGaussQuadrature
+
+makedocs(;
+    modules = [FastGaussQuadrature],
+    format = Documenter.HTML(
+        canonical = "https://JuliaApproximation.github.io/FastGaussQuadrature.jl/stable/",
+        assets = ["assets/favicon.ico"],
+    ),
+    pages = [
+        "Home" => "index.md",
+        "Gaussian Quadrature" => "gaussquadrature.md",
+        "References" => "reference.md",
+    ],
+    repo = "https://github.com/JuliaApproximation/FastGaussQuadrature.jl/blob/{commit}{path}#L{line}",
+    sitename = "FastGaussQuadrature.jl",
+    # authors = "",
```

Comment:
Question:
Who should be assigned here?

Reply:
I think @ajt60gaibb, @popsomer and myself. So

Alex Townsend, Sheehan Olver, Peter Opsomer

Reply:
I'll change it that way.

Reply:
Can I add words `and contributors` like in other repos?
https://github.com/JuliaDocs/Documenter.jl/blob/master/docs/make.jl#L25

Reply:
👍